.github/workflows/test.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - uses: actions/checkout@v2
       - id: set-modules
         run: |
-          GO_MODULES=$(find . -type f -name go.mod -exec dirname {} \; | jq -R -s -c 'split("\n")[:-1]')
+          GO_MODULES=$(find . -type d -name platform -prune -or -type f -name go.mod -exec dirname {} \; | jq -R -s -c 'split("\n")[:-1]')
```

Comment:
Made this change to avoid dealing with lint rules we inflict on ourselves but don't want to enforce on vendored code.

It means "if you find a directory (`-type d`) named "platform" then do not descend into it (`-prune`); otherwise (`-or`) find a regular file with name `go.mod` ..."